src/widgets/openmediavault/OpenMediaVaultCpu.tsx

```diff
@@ -0,0 +1,63 @@
+import { Box, Group, Paper, RingProgress, Text } from '@mantine/core';
+import { IconCpu } from '@tabler/icons-react';
+
+const OpenMediaVaultCpu = ({ info, cpuTemp }: any) => {
+  const loadData = [
+    {
+      label: 'Load Average (1min)',
```

Comment:
Use translations. Careful there are multiple parts where you didn't. Better to have every string that appears on screen translatable.